views/default/tab_malware.pdt

```diff
@@ -0,0 +1,77 @@
+<div class='title_row first'>
+    <h3><?php $this->_('Cwatch.site.malware'); ?></h3>
+</div>
+   <div id="change_password" class="col-md-6">
+        <h4><?php $this->_('Cwatch.add_new_site');?></h4>
+
+        <?php
+        print_r($sites);
+        $this->Form->create($this->base_uri . 'services/manage/' . $this->Html->ifSet($service_id) . '/tabClientMalWare/');
+        $this->Form->fieldHidden('actionname','addscanner');
+        ?>
+        <div class="col-md-12">
+            <div class="form-group">
+                <?php
+                $this->Form->label($this->_('Cwatch.domain_name', true), 'domainname');
+                $this->Form->fieldText('domainname','',['class'=>'form-control']);
+                ?>
+            </div>
+             <div class="form-group">
+                <?php
+                $this->Form->label($this->_('Cwatch.site.ftppassword', true), 'v');
```

Comment:
How come all of these labels are for a `v` field that doesn't exist? I should be able to click on any label and the cursor should be focused on that form field.